<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_music_single</class>
 <widget class="QDialog" name="add_music_single">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a song</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Single music file</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>111</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>File:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Artist:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Song:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Genre 1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Genre 2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This field will be used to see if a music is &amp;quot;national&amp;quot; or &amp;quot;international&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Country:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This date will be used to calculate the amount of new music played&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Published date:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>371</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="txt_file"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_artist"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_song"/>
    </item>
    <item>
     <widget class="QComboBox" name="cbox_g1">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(87, 118, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbox_g2">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(87, 118, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbox_country">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(87, 118, 255);</string>
      </property>
      <property name="currentText">
       <string>Portugal (or Portuguese speaking partners of CPLP)</string>
      </property>
      <item>
       <property name="text">
        <string>Portugal (or Portuguese speaking partners of CPLP)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../usr/share/icons/Mint-X/mimetypes/48/application-ogg.png</pixmap>
   </property>
  </widget>
  <widget class="QDateEdit" name="f_date">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_manageGenres">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage Genres</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
